specs/003-login-ui/tasks.md

```
@@ -0,0 +1,378 @@
+
+- [ ] T029 Final production build and validation:
+  - Build command: `npm run build`
+  - Verify zero errors, zero warnings
+  - Verify bundle size is acceptable (simple form component)
+  - Verify feature works correctly in production build (test sign-in flow)
+
+---
+
+## Dependencies & Execution Order
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)